stdpopsim/genomes.py

```diff
@@ -138,6 +139,19 @@ class Contig:
     recombination_map = attr.ib(default=None, kw_only=True)
     mutation_rate = attr.ib(default=None, type=float, kw_only=True)
     genetic_map = attr.ib(default=None, kw_only=True)
+    # TODO: figure out if ends are inclusive or not
```

Comment:
we should probably do half open intervals to match bed specification

Reply:
Left-inclusive, right-exclusive, hopefully? Matching what everything is in tskit/msprime?

Reply:
yeah, I need to be careful when gluing this to the annotations because genomic elements in SLiM are 0-indexed left and right inclusive.